index.html

```diff
@@ -40,9 +40,141 @@ <h4 class="sub-heading">
                         <i class="fa-solid fa-m social"></i>
                 
                     </div>
-                    <div class="bg-img">
-                        
-                    </div>
     </div>
+    <div class="title">
+        <h1>My Recentes Works</h1>
+        <hr>
+    </div>
+    <div class="block">
+        <img src="images/Img Placeholder.png" alt="" class="image">
+    </div>
+    <div class="block-heading">
+        <h2>Multi-Post Stories</h2>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+    </div>
+    <div class="item">
+        <div>CSS</div>
+        <div>HTML</div>
+        <div>Bootstrap</div>
+        <div>Ruby</div>
+    </div>
+    <button class="btn">See project</button>
+    <div class="block2">
+       <h1>Profesional Art <br>
+        Printing Data</h1>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+        </p>
+        <span class="card-item">
+            <div>HTML</div>
+            <div>Bootstrap</div>
+            <div>Ruby</div>
+        </span>
+        <button class="btn-2">See project</button>
+    </div>
+    <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
+         <p>
+             A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+         </p>
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
```

Comment:
Replace this h1 heading with another one as It is allow to only have one h1 tag per page please consider refractoring your code use other heading tags: h2,h3,h4,h5,h6